docs/NooBaaNonContainerized/Upgrade.md

```diff
@@ -69,5 +69,161 @@ The currently available upgrade process of NooBaa Non Containerized is an offlin
+        "upgrade_package_version":"5.18.0",
+        "in_progress_upgrade":[{
+            "timestamp":1730890691016,
+            "completed_scripts": [],
+            "running_host":"my_host1",
+            "config_dir_from_version":"0.0.0",
+            "config_dir_to_version":"1.0.0",
+            "package_from_version":"5.17.1",
+            "package_to_version":"5.18.0"
+        }]
+    }
+}
+```
+
+2. After a successful upgrade - `cat /etc/noobaa.conf.d/system.json | jq .`
+```json
+{
+  "my_host1":{
+    "current_version":"5.18.0",
+    "config_dir_version": "1.0.0",
+    "upgrade_history":{
+      "successful_upgrades":[{
+        "timestamp":1730890665481,
+        "from_version":"5.17.1",
+        "to_version":"5.18.0"
+      }]
+    }
+  },
+    "config_directory":{
+        "phase":"CONFIG_DIR_UNLOCKED",
+        "config_dir_version":"1.0.0",
+        "upgrade_package_version":"5.18.0",
+        "upgrade_history":{
+            "successful_upgrades":[{
+                "timestamp":1730890691016,
+                "completed_scripts":
+                ["/usr/local/noobaa-core/src/upgrade/nc_upgrade_scripts/1.0.0/config_dir_restructure.js"],
+                "running_host":"my_host1",
+                "config_dir_from_version":"0.0.0",
+                "config_dir_to_version":"1.0.0",
+                "package_from_version":"5.17.1",
+                "package_to_version":"5.18.0"
+            }]
+        }
+    }
+}
+```
+
+3. After a failing upgrade - `cat /etc/noobaa.conf.d/system.json | jq .`
+```json
+{
+  "my_host1":{
+    "current_version":"5.18.0",
+    "config_dir_version": "1.0.0",
+    "upgrade_history":{
+      "successful_upgrades":[{
+        "timestamp":1730890665481,
+        "from_version":"5.17.1",
+        "to_version":"5.18.0"
+      }]
+    }
+  },
+    "config_directory":{
+        "phase":"CONFIG_DIR_UNLOCKED",
+        "config_dir_version":"1.0.0",
+        "upgrade_package_version":"5.18.0",
+        "upgrade_history":{
+            "last_failure":{
+                "timestamp":1730890676741,
+                "completed_scripts":[],
+                "running_host":"my_host1",
+                "config_dir_from_version":"0.0.0",
+                "config_dir_to_version":"1.0.0",
+                "package_from_version":"5.17.1",
+                "package_to_version":"5.18.0",
+                "error":"Error: _run_nc_upgrade_scripts: nc upgrade manager failed!!!, Error: Directory not empty\n    at NCUpgradeManager._run_nc_upgrade_scripts (/usr/local/noobaa-core/src/upgrade/nc_upgrade_manager.js:253:19)\n    at async NCUpgradeManager.upgrade_config_dir (/usr/local/noobaa-core/src/upgrade/nc_upgrade_manager.js:122:13)\n    at async start_config_dir_upgrade (/usr/local/noobaa-core/src/manage_nsfs/upgrade.js:52:29)\n    at async Object.manage_upgrade_operations (/usr/local/noobaa-core/src/manage_nsfs/upgrade.js:22:13)\n    at async main (/usr/local/noobaa-core/src/cmd/manage_nsfs.js:73:13)"
```

Comment:
Could you please write the case on how we've got to this error example?

Reply:
not sure where I copied it from, I updated the examples, they had some errors